doc/python.rst

```diff
@@ -759,9 +759,11 @@ or
 
 .. code-block:: python
 
-  # pool shutdown on object deletion, interpreter shutdown, or call to rollout_.shutdown_pool
+  # pool shutdown on object deletion or call to rollout_.shutdown_pool
+  # to ensure clean shutdown of threads, call shutdown_pool before interpreter exit
```

Comment:
Please rename `shutdown_pool` to `close`, since we may want to release other resources in a future implementation, and `close()` is quite a common name for this kind of method.